src/OpenFeature/Model/EvaluationContext.cs

```diff
@@ -24,6 +24,16 @@ internal EvaluationContext(string? targetingKey, Structure content)
             this._structure = content;
         }
 
+        /// <summary>
+        /// Internal constructor used by the builder.
+        /// </summary>
+        /// <param name="content">The content of the context.</param>
+        internal EvaluationContext(Structure content)
+        {
+            this.TargetingKey = string.Empty;
+            this._structure = content;
+        }
+

```

Comment:
Is this an intended change? It seems out of scope.

Reply:
Reverted, this was unintentional. Nice catch.